.travis.yml

```diff
@@ -0,0 +1,25 @@
+language: php
+
+sudo: false
+
+cache:
+  directories:
+    - "$HOME/.composer/cache"
```

Comment:
however, I would only cache ``"$HOME/.composer/cache/files". no need to persist the cache for data downloaded from Packagist: the packages used by the ACL component are active ones and so the cache will miss on most builds, forcing Travis to store an updated cache each time.